### PR TITLE
Documentation is on Read the Docs

### PR DESCRIPTION
**Pull request by jakirkham**
Updates all info about documentation to point to Read the Docs.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -475,7 +475,7 @@ def setup_package():
         'description': (
             'A pythonic wrapper around FFTW, the FFT library, presenting a '
             'unified interface for all the supported transforms.'),
-        'url': 'http://pyFFTW.github.io/pyFFTW/',
+        'url': 'https://github.com/pyFFTW/pyFFTW',
```
Comment (jakirkham) [inline setup.py]: The alternative to this would be to link to Read the Docs here as well.

**Comment by jakirkham** [comment]
Would add that I think we should also update the link in the repo description too. Perhaps it should point to Read the Docs also. Thoughts?

**Comment by hgomersall** [comment]
This looks great, except I'm not sure we have all the versions set up properly? Does RTD build for each branch? In which case we should branch off 0.10.4 (or whatever is on pypi currently) and build for that too (as well as future releases).

**Comment by hgomersall** [comment]
This is easy to merge. Thanks.

**Pull request merged**